src/Rules/Resources/ConditionDescriptions.resx

```diff
@@ -131,7 +131,7 @@
   </data>
   <data name="BoundingRectangleCompletelyObscuresContainer" xml:space="preserve">
     <value>BoundingRectangle completely obscures container</value>
-    <comment>The name of a condition used by the automated accessibility scanner. The term "BoundingRectangle" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <comment>The name of a condition used by the automated accessibility scanner. The term "BoundingRectangle" should not be translated (i.e. it should be treated as a loanword). {Locked="BoundingRectangle"}</comment>
```

Comment:
Can we omit the human-readable equivalent of the locked keyword?
```suggestion
    <comment>The name of a condition used by the automated accessibility scanner.{Locked="BoundingRectangle"}</comment>
```

If you agree, we should follow the same pattern for all strings.

Reply:
Even though it makes the comments longer, I feel like maintaining both human- and machine-readable comments here only aids clarity.

Reply:
Discussed offline, changes to follow.